scripts/docker/build_docker.py

```diff
@@ -72,10 +72,11 @@ class ImageDependencies:
                            that this dockerfile derives from, and the values indicate corresponding ARG values that
                            must be passed to docker_build to specify the correct repo/image:tag
     """
+
     def __init__(
-            self,
-            git_dependencies: Union[str, Iterable[str]],
-            docker_dependencies: Mapping[str, str] = MappingProxyType({})
+        self,
```

Comment:
Should we suggest using black upstream to mitigate conflicts in the future?

Reply:
I'll raise this on their upstream repo